products/cloudidentity/terraform.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Group: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_identity_groups_basic"
+        primary_resource_id: "cloud_identity_group_basic"
+        min_version: beta
+        vars:
+          id_group: "my-identity-group"
+        test_env_vars:
+          org_domain: :ORG_DOMAIN
+          cust_id: :CUST_ID
+      ### The full API doesn't seem to be implemented yet
+      # - !ruby/object:Provider::Terraform::Examples
+      #   name: "cloud_identity_groups_full"
+      #   primary_resource_id: "cloud_identity_group_full"
+      #   min_version: beta
+      #   vars:
+      #     id_group: "my-identity-group"
+      #   test_env_vars:
+      #     org_domain: :ORG_DOMAIN
+      #     cust_id: :CUST_ID
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      post_create: templates/terraform/post_create/set_computed_name.erb
+  GroupMembership: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["{{name}}"]
+    examples:
```

Comment:
Even though it's really similar, I think people would probably appreciate an example showing how to add a user in addition to the existing one for adding a child group.

Reply:
I agree, I started with that, but the user has to exist, and I can't create that with terraform, I don't believe.  Do you want me to put an example in that is `skip_test: true` and hard-code an example email?

Reply:
Does the user have to be in the org in question? If not I think we have a bunch of tests that use various emails belonging to people (like Paddy)

Reply:
Yea, it has to end with the org_domain.

Reply:
I put the example in with `skip_test`, I tested it first with a user I created in our org, and it worked, so I'm confident it will work.